script.js

```diff
@@ -0,0 +1,121 @@
+
+var canvas = document.getElementById("canvas");
+var ctx = canvas.getContext("2d");
+var radius = canvas.height / 2;
+ctx.translate(radius, radius);
+radius = radius * 0.90
+setInterval(drawClock, 1000) //Analog Clock
+currentTime();  // Digital Clock
+
+// Drawing the Clock;s canvas and face
+function drawClock() {
+    drawFace(ctx, radius);
+}
+
+function drawFace(ctx, radius) {
+    var grad;

```

Comment:
I think you can do something like var grad = ctx.createRadialGradient below, you don't need to declare at the top